src/components/AppSettingsMenu.vue

```diff
@@ -4,14 +4,14 @@
 			:name="t('mail', 'Mail settings')"
 			:show-navigation="true"
 			:open.sync="showSettings">
-			<NcAppSettingsSection id="mail-list-view" :name="t('mail', 'Layout')">
+			<NcAppSettingsSection v-if="!isMobile" id="mail-list-view" :name="t('mail', 'Layout')">
 				<NcCheckboxRadioSwitch value="no-split"
 					:button-variant="true"
 					name="mail-layout"
 					type="radio"
 					:checked="layoutMode"
 					button-variant-grouped="vertical"
-					@update:checked="setLayout">
+					@update:checked="() => setLayout('no-split')">
```

Comment:
```suggestion
					@update:checked="setLayout('no-split')">
```

this works the same, no?